src/quo/components/separator.cljs

```diff
@@ -0,0 +1,12 @@
+(ns quo.components.separator
+  (:require [status-im.ui.components.react :as react]
```

Comment:
We have `quo.react-native` namespace to make quo independent of stat-im namespaces.